### DISCUSSION TITLE
Audio humming in DK64

### DISCUSSION THREAD

**Discussion by Seanboy123** [mupen64plus-ae/mupen64plus-ae#1154] [Q&A]
Attached will be a link to the screen recording a took. Only started happening in during Gloomy Galleon boss fight and won't stop.[Audio and video clip](https://youtu.be/U91Y9SOxEyA?si=DW9zxD1QGxcymSp6)

**Comment by fzurita** [comment]
This is an issue with using save states in that game. Save states in that game also corrupt in-game saves.

The only solution is to always use in-game saves and never load any save states.
